### PR TITLE
Update text_unit to text-size

### PR DESCRIPTION
**Pull request by CAD97**

### CODE REVIEW

File: src/green/element.rs [hunk 1]
```diff
@@ -63,11 +63,12 @@ impl GreenElement {
     pub fn kind(&self) -> SyntaxKind {
         self.as_ref().kind()
     }
+}
 
-    /// Returns length of the text covered by this element.
+impl TextLen for &'_ GreenElement {
     #[inline]
-    pub fn text_len(&self) -> TextUnit {
```
Comment (matklad) [inline src/green/element.rs:69]: This should be kept as a public inherent method and used instead of `TextUnit::of(&node)`. Otherwise, r=me

Comment (matklad) [reply src/green/element.rs:69]: Oh, let's allso change the version of the crate itself to `0.next.0-pre`, so that I can publish it?

Comment (CAD97) [reply src/green/element.rs:69]: Did so.

**Pull request merged**